### PR TITLE
40 remove contract inputs and outputs from tx list preview

### PR DESCRIPTION
**Pull request by QuinnLee**
Screen Shot 2022-04-01 at 12 25 42 AM

I also updated prettier to check `tsx` files
## UPDATED 
Screen Shot 2022-04-01 at 9 52 58 AM

### CODE REVIEW

File: src/pages/HomePage/RecentTransactions.tsx [hunk 1]
```diff
@@ -87,15 +79,20 @@ function TransactionRow({ transaction }: { transaction: HomePageTransaction }) {
...
-          <TransactionRecipientLabel>To:</TransactionRecipientLabel>
-          {transaction.outputs.map((output) =>
-            output.__typename === "CoinOutput" ? (
-              <TransactionRecipientLink to={`/address/${output.to}`}>
-                {output.to}
-              </TransactionRecipientLink>
-            ) : (
-              output.__typename
-            )
+          {transaction.outputs.map(
```
Comment (QuinnLee) [inline src/pages/HomePage/RecentTransactions.tsx:82]: There's got to be an easier way. Are we using fuels TS here???

**Review by adlerjohn — COMMENTED** [review]
Wait, there's still a "From: InputContract" in the list!

**Comment by QuinnLee** [comment]
Screen Shot 2022-04-01 at 9 25 55 AM
 
Updated UI

**Comment by adlerjohn** [comment]
> I can also add the type so it looks like

That would be good just to make it uniform with outputs.

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.4.13",
-    "@craco/craco": "^6.4.3",
+    "@craco/craco": "7.0.0-alpha.3",
```
Comment (luizstacio) [inline package.json:37]: Why this bump?

File: src/pages/HomePage/RecentTransactions.tsx (hunk 1 above)
Comment (luizstacio) [reply src/pages/HomePage/RecentTransactions.tsx:84]:
```ts
{transaction.outputs.map(
  (output) =>
    ['ChangeOutput', 'CoinOutput', 'VariableOutput', 'WithdrawalOutput'].contains(output.__typename) && (
      <>
        <TransactionRecipientLabel>To:</TransactionRecipientLabel>
        {output.__typename}
        <TransactionRecipientLink to={`/address/${output.to}`}>
          {output.to}
        </TransactionRecipientLink>
      </>
    )
)}
```

**Review by luizstacio — CHANGES_REQUESTED** [review]
Overall looks! good only some comments

File: package.json (hunk 2 above)
Comment (QuinnLee) [reply package.json:37]: I'm trying to resolve it a build issue https://github.com/FuelLabs/block-explorer-v2/runs/5788640372?check_suite_focus=true

It doesn't make sense.  I know CRACO doesn't support `react scripts v5`, but why did the CI start breaking?

Comment (luizstacio) [reply package.json:37]: I made some tests with the branch master and this is due to an incorrect update `package-lock.json` it was working. I had to remove the `package-lock` on the master locally and try to install it and got the same error.

Nice...

Comment (QuinnLee) [reply package.json:37]: I tried to downgrade react scripts, but it's a bit of a mess because of the TS version.  Can we keep this version of craco?

**Review by luizstacio — APPROVED** [review]

**Pull request merged**